### PR TITLE
Feature: install matter python requirements (VSC-1030)

### PR DESCRIPTION
**Pull request by Diegorro98**
## Description

Installs python requirements for ESP-Matter when it gets installed.
Also added a command to install this requirements in other python environments in case that is needed.

And fixed some notifications in ESP-Matter installation.

Fixes # ([VSC-1029](https://github.com/espressif/vscode-esp-idf-extension/issues/868))

## Type of change

Please delete options that are not relevant.
- New feature (non-breaking change which adds functionality)

## How has this been tested?

Build the extension and run the whole installation process. Also I tested the command in another python env.

**Test Configuration**:
* ESP-IDF Version: 4.4.3
* OS (Windows,Linux and macOS): Linux

## Checklist
- [x] PR Self Reviewed
- [x] Applied Code formatting
- [ ] Added Documentation
- [ ] Added Unit Test
- [ ] Verified on all platforms - Windows,Linux and macOS

### CODE REVIEW

**Comment by Diegorro98** [comment]
Formatting rebase

File: src/espMatter/espMatterDownload.ts [hunk 1]
```diff
@@ -16,7 +16,7 @@
  * limitations under the License.
  */
 import { pathExists } from "fs-extra";
-import { join } from "path";
+import path, { join } from "path";
```
Comment (brianignacio5) [inline src/espMatter/espMatterDownload.ts:19]: Need to fix this import here.

**Review by brianignacio5 — APPROVED** [review]
LGTM with minor fix request

**Comment by Diegorro98** [comment]
@brianignacio5 Thanks for reviewing! Fixed the import in last rebase.

**Pull request merged**